packages/aws-client/src/S3.ts

```diff
@@ -1063,7 +1065,8 @@ export const moveObject = async (
     destinationBucket: string,
     destinationKey: string,
     ACL?: AWS.S3.ObjectCannedACL,
-    copyTags?: boolean
+    copyTags?: boolean,
+    maxChunkSize?: number
```

Comment:
Yeah I think we should name this chunkSize unless you can give me a good reason not to?